packages/types/types/index.d.ts

```diff
@@ -83,6 +112,12 @@ export interface CardProps
   borderWidth?: 0 | 1 | 2
 }
 
+export interface CloseButtonProps
+  extends IconButtonProps,
+  ButtonProps {
+  title?: string
```

Comment:
I think `CloseButton` also needs `size?: number`

Side note: that's weird, because it also passes that size prop along to `IconButton`, so it seems like `size` is being used in two different ways by that component.

Reply:
Yeah you're right, `CloseButton` uses its `size` prop to control the Close icon's size. It's unfortunate because I have to add `number` as a type to the `ButtonProps` interface otherwise it conflicts with the inherited type.

```
Interface 'CloseButtonProps' incorrectly extends interface 'ButtonProps'.
  Types of property 'size' are incompatible.
    Type 'number | undefined' is not assignable to type '"small" | "medium" | "large" | undefined'.
      Type 'number' is not assignable to type '"small" | "medium" | "large" | undefined'.ts(2430)
```

Perhaps we should add an issue to refactor `CloseButton`'s size prop to be more specific like `iconSize` so that it's less muddy?